dspace-api/src/main/java/org/dspace/content/authority/DSpaceControlledVocabulary.java

```diff
@@ -132,11 +132,15 @@ protected void init() {
             String filename = vocabulariesPath + vocabularyName + ".xml";
             log.info("Loading " + filename);
             vocabulary = new InputSource(filename);
+
+            hierarchyDelimiter = " " + hierarchyDelimiter + " ";
```

Comment:
How does this simple change impact sites which are already using these subjects?  I noticed that prior to this PR, a `dc.subject` value would be something like `HUMANITIES and RELIGION::Religion/Theology`.  But, after this PR, selecting the same subject sets the `dc.subject` equal to `HUMANITIES and RELIGION :: Religion/Theology`

Do we need a way to migrate from the old format to the new one?

Reply:
This change was just done for the sake of making it more readable. 
This would however have as effect that old repositories would need to migrate to the new format (by using an sql query for example).

For now I just reverted the change [here](https://github.com/DSpace/DSpace/commit/bf084d74a1e7a6271240599b7e0ed800170894c8).